ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java

```diff
@@ -0,0 +1,37 @@
+package com.assamblypayments.ramenpos;
+
+public class Enums {
+    static final String Pair = "Pair";
```

Comment:
Constants in Java are named in `CAPITAL_CASE`, not camel case.

Reply:
Done